include/opendht/dht_proxy_server.h

```diff
@@ -73,15 +77,46 @@ class OPENDHT_PUBLIC DhtProxyServer
         size_t pushListenersCount;
         /** Average requests per second */
         double requestRate;
+        /** Node Info **/
+        NodeInfo nodeInfo;
 
         std::string toString() const {
             std::ostringstream ss;
             ss << "Listens: " << listenCount << " Puts: " << putCount << " PushListeners: " << pushListenersCount << std::endl;
             ss << "Requests: " << requestRate << " per second." << std::endl;
+            auto& ni = nodeInfo;
+            auto& ipv4 = ni.ipv4;
+            if (ipv4.table_depth > 1) {
+                unsigned long tot_nodes = 8 * std::exp2(ipv4.table_depth);
```

Comment:
Add method getNetworkSizeEstimation to NodeStats to factorize computation

Reply:
Done